src/rsz/src/SteinerTree.cc

```diff
@@ -32,6 +32,40 @@ static void connectedPins(const Net* net,
                           // Return value.
                           Vector<PinLoc>& pins);
 
+SteinerTree* Resizer::makeSteinerTree(Point drvr_location,
+                                      const std::vector<Point>& sink_locations)
+{
+  SteinerTree* tree = new SteinerTree(drvr_location, this);
+  Vector<PinLoc>& pinlocs = tree->pinlocs();
+  for (auto loc : sink_locations) {
+    pinlocs.push_back(PinLoc{nullptr, loc});
+  }
+  // Sort pins by location
+  sort(pinlocs, [=](const PinLoc& pin1, const PinLoc& pin2) {
+    return pin1.loc.getX() < pin2.loc.getX()
+           || (pin1.loc.getX() == pin2.loc.getX()
+               && pin1.loc.getY() < pin2.loc.getY());
+  });
+  int pin_count = pinlocs.size();
+  if (pin_count >= 1) {
+    std::vector<int> x,
+        y;  // Two separate vectors of coordinates needed by flute.
```

Comment:
Icky formatting, put the comment on the line above

Reply:
Done